sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -10,6 +10,7 @@ dayjs.extend(customParseFormat);
 
 const DEFAULT_EMPTY_VALUE = "-";
 export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
+export const HYPHEN_WITH_SPACE = " - ";
```

Comment:
Please see https://github.com/bcgov/SIMS/pull/1914#discussion_r1184417241

Reply:
removed it